packages/k/kitty/abi_used_libs

```diff
@@ -9,8 +9,8 @@ libharfbuzz.so.0
 liblcms2.so.2
 libm.so.6
 libpython3.10.so.1.0
-librsync.so.2
```

Comment:
Why does it no longer link against librsync? Nothing in the changelog seems to be related to this change.

If the library is no longer necessary the devel version of it should be dropped from the builddeps.